docs/section6/Rule6-10.md

```diff
@@ -0,0 +1,65 @@
+
+# Lighting - Rule 6-10  
+
+**Rule ID:** 6-10  
+**Rule Description:** Baseline building is modeled with occupancy sensor controls in applicable space types, i.e. employee lunch and break rooms, conference/meeting rooms, and classrooms (not including shop classrooms, laboratory classrooms, and preschool through 12th-grade classrooms).  
+**Appendix G Section Reference:** Section G3.1-6 Modeling Requirements for the baseline building  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+**Function Call:**  
+
+  - compare_schedules()
+  - normalize_space_lighting_schedule()
+
+## Rule Logic: 
+
+- Get building open schedule in the baseline model: `building_open_schedule_b = B_RMR.building.building_open_schedule`  
+
+- For each building segment in building: `building_segment_b in B_RMR.building.building_segments:`  
+
+  - For each thermal block in building segment: `thermal_block_b in building_segment_b.thermal_blocks:`  
+
+    - For each zone in thermal block: `zone_b in thermal_block_b.zones:`  
+
+      - For each space in zone: `space_b in zone_b.spaces:`  
+
+        - Get interior lighting in space: `interior_lighting_b = space_b.interior_lighting`  
+
+          - Get interior lighting occupancy control type: `control_type_b = interior_lighting_b.occupancy_control_type`  
+
+        - Get normalized space lighting schedule: `normalized_schedule_b = normalize_space_lighting_schedule(space_b)`  
+
+        - Get matching space in P_RMR: `space_p = match_data_element(P_RMR, Spaces, space_b.id)`  
+
+          - Get normalized space lighting schedule: `normalized_schedule_p = normalize_space_lighting_schedule(space_p)`  
+
+        - Compare lighting schedule in B_RMR and P_RMR: `schedule_comparison_result = compare_schedules(normalized_schedule_b, normalized_schedule_p, building_open_schedule_b, 1)`  
+
+        - Get lighting space type: `lighting_space_type_b = space_b.lighting_space_type`  
+
+          - If lighting space types is employee lunch and break rooms, conference/meeting rooms, or classrooms (not including shop classrooms, laboratory classrooms, and preschool through 12th-grade classrooms), set occupancy sensor flag: `if lighting_space_type_b in ["LOUNGE/BREAKROOM", "CONFERENCE/MEETING/MULTIPURPOSE ROOM", "CLASSROOM/LECTURE HALL/TRAINING ROOM - PENITENTIARY", "CLASSROOM/LECTURE HALL/TRAINING ROOM - ALL OTHER"]: occ_sensor_flag = TRUE`  
+
+          **Rule Assertion:**  
+
+          - Case 1: If space requires occupancy sensor and interior lighting is modeled with full-auto on occupancy sensor control: `if ( occ_sensor_flag ) and ( control_type_b == "FULL-AUTO ON" ): PASS and return schedule_comparison_result`  
+
```

Comment:
I think here we just need to establish that normalized schedules are the same for baseline and proposed 24/7. So I don't think we need to check for control_type. I would just check if normalized schedules are the same for the baseline and proposed, and have PASS if yes and FAIL if no.

We'll need to revisit once "schedule_compare" function is finalized so that we know what it returns.